pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
+    rpc QueryPages(IndexQuery) returns (ReadBatch);
+
+    //storage-client
+    rpc PutChunks(Chunk) returns (Nothing);
+    rpc GetChunks(Chunks) returns (Chunks);
+    rpc DeleteChunks(ChunkID) returns (Nothing);
+
+    //table-client
+    rpc ListTables(Nothing) returns (ListTablesResponse);
+    rpc CreateTable(TableDesc) returns (Nothing);
+    rpc DeleteTable(TableName) returns (Nothing);
+    rpc DescribeTable(TableName) returns (DescribeTableResponse);
+    rpc UpdateTable(UpdateTableRequest) returns (Nothing);
+
+    //used by storage-client & index-client
+    rpc Stop(Nothing) returns (Nothing);
+}
+
+message Nothing {}
+
+message ChunkID {
+    string chunkId = 1;
+}
+
+message TableName {
+    string tableName = 1;
+}
+
+message WriteBatch {
+    repeated IndexEntry indexEntry = 1;
+}
+
+message ReadBatch {
```

Comment:
Why is the ReadBatch only returning a single row? We should accept multiple rows in the ReadBatch.

Reply:
Changes this to stream. So now ReadBatch sends multiple rows in the single RPC call